backend/src/main/java/com/bakdata/conquery/models/auth/permissions/Ability.java

```diff
@@ -8,15 +8,18 @@
 	READ,
 	CREATE,
 
-	// Query Specific
+	// Query specific
 	TAG,
 	CANCEL,
 	LABEL,
 	SHARE,
 
-	// Dataset Specific
+	// Dataset specific
 	DOWNLOAD,
-	PRESERVE_ID
+	PRESERVE_ID,
+	
+	// FormConfig specific
+	MODIFY
```

Comment:
🤔 😬

Reply:
Just curious: Why is a new permission necessary? If it really is, how about calling it MODIFY_FORM_CONFIG and removing the comment?

Reply:
The new Permission is the FormConfigPermission. This is just one of the abilities, that is available for that type.
A stricter differentiation between Meta-Data of an object (creation.-time, tags, ...) and the actual content might be a better approach to also simplify the Permission and their abilities.

Reply:
Okay, so: Why is the new "FormConfigPermission" necessary?

And about the naming of the abilities here, it seems a little odd that we have TAG LABEL SHARE for queries, but for formConfigs it's called `MODIFY`, no?

Reply:
We have a Permission for each type of protected Resource (Query, Dataset, Formular ...).
The abilities you mentioned also apply to the FormConfigPermission, those were initially only for the QueryPermission. However the possiblity to modify the content of an Query does not exsists at the moment.
The Abilities are probably a bit to fine grain at this point.